### DISCUSSION TITLE
DMG files

### DISCUSSION THREAD

**Discussion by gisms** [sepinf-inc/IPED#1526] [General]
Hi!

We did a mac extraction in DMG. How do you usually deal in this case to index via iped?

**Comment by lfcnassif** [comment]
Try to convert it to a supported format (dd/e01) using FTKImager, it usually works.

**Reply by gisms** [reply]
We tried to open the dmg directly through FTK, but it gave an error. Would this conversion via ftk be through mounting the dmg before and later generating an imgae from the mounted drive?

**Reply by lfcnassif** [reply]
No, it's like generating a new image from the existing one, but using another destination image format. If FTKImager returned error when opening the DMG, maybe it is corrupted or the DMG version is not supported, try updating FTKImager.

If the error was related to unrecognized/unknown file system, the image conversion procedure should work, and perhaps IPED/TSK will be able to recognize the file system, they use a different implementation (but usually FTKImager FS detection/decoding is better than Sleuthkit).

**Reply by gisms** [reply]
ok, thank you a lot!!!
